Framework/API/inc/MantidAPI/IMDHistoWorkspace.h

```diff
@@ -44,6 +44,10 @@ class DLLExport IMDHistoWorkspace : public IMDWorkspace,
   IMDHistoWorkspace_uptr clone() const {
     return IMDHistoWorkspace_uptr(doClone());
```

Comment:
No, we *need* covariance, i.e., subtypes of `IMDHistoworkspace` must return a `unique_ptr` to their own type, not to the base class type.